src/SlashCommands.js

```diff
@@ -110,6 +110,24 @@ export const CommandMap = {
         },
     }),
 
+    roomnick: new Command({
+        name: 'roomnick',
+        args: '<display_name>',
+        description: _td('Changes your display nickname in the current room only'),
+        runFn: function(roomId, args) {
+            if (args) {
+                const cli = MatrixClientPeg.get();
+                const ev = cli.getRoom(roomId).currentState.getStateEvents('m.room.member', cli.getUserId());
+                const content = {
+                    ...ev ? ev.getContent() : null,
```

Comment:
`getContent()` will always return an object, however `ev` may be null (because cache problems) - we should null guard that

Reply:
That's what that ternary is doing though, no?

Reply:
oh, yup, that's a thing.

There is the scenario where we'd end up sending an event without a `membership` key though - it should probably always be set to 'join'.